web/src/js/ducks/options.js

```diff
@@ -6,7 +6,11 @@ export const RECEIVE = 'OPTIONS_RECEIVE'
 export const UPDATE = 'OPTIONS_UPDATE'
 export const REQUEST_UPDATE = 'REQUEST_UPDATE'
 
-const defaultState = {}
+const defaultState = {
+    columns: {
+        value: ["", "Path", "Method", "Status", "Size", "Time"]
+    }
```

Comment:
Let's keep this empty here (and just have the FlowRow fall back to defaults if the option is not set)